src/drivers/imu/adis16477/ADIS16477.cpp

```diff
@@ -71,17 +81,11 @@ static constexpr uint16_t PROD_ID_ADIS16477 = 0x405D; /* ADIS16477 Identificatio
 
 static constexpr int T_STALL = 16;
 
-#define GYROINITIALSENSITIVITY		250
-#define ACCELINITIALSENSITIVITY		(1.0f / 1200.0f)
-#define ACCELDYNAMICRANGE		18.0f
```

Comment:
These values are unused at present.  The modified values in this PR are taken from the datasheet.  The difference in values has no functional impact, and the datasheet values are being put in place here to prevent any future issues.